layers/timeline_semaphore.c

```diff
@@ -1195,6 +1202,13 @@ static VkResult timeline_SignalSemaphore(
     return result;
 }
 
+static VkResult timeline_SignalSemaphore(
```

Comment:
This doesn't compile : 
```
[ 33%] Building C object layers/CMakeFiles/VkLayer_khronos_timeline_semaphore.dir/timeline_semaphore.c.o
Vulkan-ExtensionLayer/layers/timeline_semaphore.c:1205:17: error: redefinition of ‘timeline_SignalSemaphore’
 1205 | static VkResult timeline_SignalSemaphore(
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~
Vulkan-ExtensionLayer/layers/timeline_semaphore.c:1167:17: note: previous definition of ‘timeline_SignalSemaphore’ was here
 1167 | static VkResult timeline_SignalSemaphore(
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~
Vulkan-ExtensionLayer/layers/timeline_semaphore.c: In function ‘timeline_SignalSemaphore’:
Vulkan-ExtensionLayer/layers/timeline_semaphore.c:1209:12: warning: implicit declaration of function ‘timeline_SignalSemaphoreKHR’; did you mean ‘timeline_SignalSemaphore’? [-Wimplicit-function-declaration]
 1209 |     return timeline_SignalSemaphoreKHR(_device, pSignalInfo);
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~
      |            timeline_SignalSemaphore
```

Reply:
Right, it's fixed by other commit. Please just review "Files Changed" instead of commit by commit. You can just squash all of the commit while they get merged.